maquetado/scss/details.scss

```diff
@@ -0,0 +1,25 @@
+.book-details {
+  p {
```

Comment:
Don't use the tag to define props. Add class to the tags and use it here.

Reply:
Done